scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1908,6 +1909,15 @@ class Router(formatOps: FormatOps) {
             Split(Newline, 1).withIndent(2, lastToken, After)
           )
         }
+      // type bounds in type definitions and declarations
+      case FormatToken(_: T.Subtype, _, _)
```

Comment:
not sure if the language allows... can the split be before the `<:` symbol? (it works like an `extends`, and we break before `extends`.) would it look nice?

Reply:
This would be something like:
```scala
type Concat[X <: Tuple, Y <: Tuple]
  <: Tuple = X match {
    case Unit =>
      Y
    case x1 *: xs1 =>
      x1 *: Concat[xs1, Y]
  }
```
Looks also ok I think. Should we change it?

Reply:
1. does it compile?

2. if the definition is long and need to have another break (say, after `&`), will the next line be indented further? (examples below _do not_ show the additional indent but could.)

3. do you like it yourself? would you prefer A, B or C? (i don't like B, i think.)

```
// A
type Concat[X, Y] <:
  Tuple & Shmuple &
  B = X match {
    ...
  }
// B
type Concat[X, Y]
  <: Tuple & Shmuple &
  B = X match {
    ...
  }
// C
type Concat[X, Y]
  <: Tuple & Shmuple
  & B = X match {
    ...
  }
```

Reply:
Ok, so I think a bit modified C makes sense here (that's what I changed to):
```scala
type Concat[X, Y]
  <: Tuple & Shmuple
    & B = X match {
      ...
    }
```
what do you think? It does compile from what I checked.